packages/ui5-application-writer/templates/optional/typescript/.babelrc.json

```diff
@@ -4,7 +4,8 @@
         [
             "transform-ui5",
             {
-                "addControllerStaticPropsToExtend": true
+                "addControllerStaticPropsToExtend": true,
```

Comment:
`addControllerStaticPropsToExtend` shouldn't be required anymore if we use `overridesToOverride`

Reply:
Thanks. Updated in https://github.com/SAP/open-ux-tools/pull/1020/commits/9838a2cd8c9277431da31417abb374eedd3c1355